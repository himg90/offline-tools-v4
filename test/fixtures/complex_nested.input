{"level1":"eyJsZXZlbDIiOiJleUpzWlhabGJETWlPaUpRYkdGcGJpQjBaWGgwSUdGMElHeGxkbVZzSURNaUxDSmxibU52WkdWa0lqb2lRbUZ6WlRZMElHRjBJR3hsZG1Wc0lETWlmUT09IiwicGxhaW5BdExldmVsMiI6IlBsYWluIHRleHQgYXQgbGV2ZWwgMiJ9","plainAtLevel1":"Plain text at level 1"} 