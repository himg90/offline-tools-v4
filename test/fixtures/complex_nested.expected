{
  "level1": {
    "level2": {
      "level3": "Plain text at level 3",
      "encoded": "Base64 at level 3"
    },
    "plainAtLevel2": "Plain text at level 2"
  },
  "plainAtLevel1": "Plain text at level 1"
} 